rocketmq-doc/_docs/zh/07-01-broker-naming.md

```
@@ -0,0 +1,171 @@
+{brokerClusterName}-{brokerId}-{instanceId}
+```
+
+但实际上，最常见和推荐的格式是：
+
+```
+{brokerClusterName}-{brokerId}
+```
+
+#### 各部分含义：
+
+| 字段                  | 说明                | 示例                   |
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)